res/controllers/Denon-Prime-4-scripts.js

```diff
@@ -1,17 +1,80 @@
 // eslint-disable-next-line-no-var
 var Prime4 = {};
 
+/*
+ * DENON DJ PRIME 4 - CONTROLLER MAPPING FOR MIXXX
+ *
+ * Thank you for using this mapping! Below are a few parameters
+ * you can change if you want to customize anything :)
+ */
+
+// What colour would you like each deck to be?
+// Choose between "red", "yellow", "green", "cyan", "blue", "magenta", or "white".
+const colourOfDeck1 = "white";
+const colourOfDeck2 = "yellow";
+const colourOfDeck3 = "cyan";
+const colourOfDeck4 = "magenta";
+
+/*
+ *                   WARNING!!!
+ *
+ *      DO NOT EDIT ANYTHING PAST THIS POINT
+ *       UNLESS YOU KNOW WHAT YOU'RE DOING.
+ */
+
+Prime4.rgbCode = {
+    black: 0,
+    blueDark: 1,
+    blueDim: 2,
+    blue: 3,
+    greenDark: 4,
+    cyanDark: 5,
+    greenDim: 8,
+    cyanDim: 10,
+    green: 12,
+    cyan: 15,
+    redDark: 16,
+    magentaDark: 17,
+    yellowDark: 20,
+    whiteDark: 21,
+    redDim: 32,
+    magentaDim: 34,
+    yellowDim: 40,
+    whiteDim: 42,
+    red: 48,
+    magenta: 51,
+    orange: 56,
+    yellow: 60,
+    white: 63,
+};
+
+const colDeck1 = eval("Prime4.rgbCode." + colourOfDeck1);
```

Comment:
**Please** don't use `eval`. JS has the bracket operator for this purpose. 
```suggestion
const colDeck1 = Prime4.rgbCode[colourOfDeck1]);
```

Reply:
Awesome thank you. I was wondering what a suitable alternative to `eval` was.